include/boost/geometry/strategies/cartesian/side_by_triangle.hpp

```diff
@@ -80,11 +81,16 @@ public :
         return envelope_strategy_type();
     }
 
-    typedef strategy::disjoint::segment_box disjoint_strategy_type;
+    typedef disjoint::segment_box
+    <
+        CalculationType,
+        distance::pythagoras<CalculationType>
+    > disjoint_segment_box_strategy_type;
```

Comment:
So side strategy now provides `disjoint::segment_box` which requires distance strategy? Why is this the case?

Reply:
disjoint::segment_box does not require distance strategy